webextensions/_locales/fr/messages.json

```diff
@@ -1,46 +1,46 @@
 {
   "extensionName": { "message": "Tree Style Tab" },
-  "extensionDescription": { "message": "Afficher les onglets dans une arborescence." },
+  "extensionDescription": { "message": "Affiche les onglets sous forme d'arbre." },
 
   "sidebarTitle": { "message": "Tree Style Tab" },
-  "sidebarToggleDescription": { "message": "Activer/Désactiver \"Tree Style Tab\" Panneau" },
+  "sidebarToggleDescription": { "message": "Activer/Désactiver le panneau « Tree Style Tab »" },
 
   "command_indent":                { "message": "Indenter l'onglet courant" },
-  "command_outdent":               { "message": "Outdent Current Tab" },
+  "command_outdent":               { "message": "Désindenter Current Tab" },
   "command_tabMoveUp":             { "message": "Déplace l'onglet courant vers le haut" },
-  "command_treeMoveUp":            { "message": "Déplace l'arborescence courante vers le haut" },
+  "command_treeMoveUp":            { "message": "Déplace l'arbre courant vers le haut" },
   "command_tabMoveDown":           { "message": "Déplace l'onglet courant vers le bas" },
-  "command_treeMoveDown":          { "message": "Déplace l'arborescence courante vers le bas" },
-  "command_focusPrevious":         { "message": "Focus sur l'onglet précédent (déplie l'arborescence)" },
-  "command_focusPreviousSilently": { "message": "Focus sur l'onglet précédent (ne déplie pas l'arborescence)" },
-  "command_focusNext":             { "message": "Focus sur l'onglet suivant (déplie l'arborescence)" },
-  "command_focusNextSilently":     { "message": "Focus sur l'onglet suivant (ne déplie pas l'arborescence)" },
+  "command_treeMoveDown":          { "message": "Déplace l'arbre courant vers le bas" },
+  "command_focusPrevious":         { "message": "Focus sur l'onglet précédent (déplie l'arbre)" },
```

Comment:
Maybe use `ouvrir` instead of `déplier`

Reply:
Throughout the file, «replier» and «déplier» are used so I think it’s the right choice here.